apps/website/src/components/preview-page.tsx

```diff
@@ -78,70 +80,76 @@ export function PreviewPage(props: PreviewPageProps) {
-                      size={80}
-                    />
-                  )}
+                  <div className="mb-2 xl:mb-4">
+                    {/* <div className="aspect-square w-20 rounded-full bg-gray-300"></div> */}
+                    {type === 'community' && (
+                      <Avatar
+                        type="community"
+                        name={verifiedData.displayName}
+                        src="https://images.unsplash.com/photo-1518020382113-a7e8fc38eac9?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=500&h=500&q=80"
+                        size={80}
+                      />
+                    )}
+                    {type === 'channel' && (
+                      <Avatar
+                        type="channel"
+                        emoji={verifiedData.emoji!}
+                        size={32}
+                      />
+                    )}
+                    {type === 'profile' && (
+                      <Avatar
+                        type="account"
+                        name={verifiedData.displayName}
+                        src="https://images.unsplash.com/photo-1518020382113-a7e8fc38eac9?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=500&h=500&q=80"
+                        size={80}
+                      />
+                    )}
+                  </div>
 
                   <h1 className="mb-3 text-4xl font-bold text-gray-900 xl:text-6xl">
                     {type === 'channel' && '#'} {verifiedData.displayName}
                   </h1>
-                  <p className="mb-4 text-[15px] text-gray-900 xl:text-[19px]">
+                  <p className="mb-3 text-[15px] text-neutral-100 xl:text-[19px]">
                     {verifiedData.description}
                   </p>
 
                   {type === 'community' && (
                     <>
-                      <div className="mb-5 flex items-center gap-1">
+                      <div className="flex items-center gap-1">
                         <MembersIcon size={20} />
-                        <span className="text-[15px]">446.7K</span>
+                        <Text size={15}>
+                          {formatNumber(verifiedData.membersCount)}
+                        </Text>
                       </div>
-
-                      <div className="flex gap-3">
-                        <div className="h-8 rounded-full bg-gray-200 px-3 text-sm">
-                          Art
-                        </div>
-                        <div className="h-8 rounded-full bg-gray-200 px-3 text-sm">
-                          Crypto
-                        </div>
-                        <div className="h-8 rounded-full bg-gray-200 px-3 text-sm">
-                          NFT
+                      {verifiedData.tags?.length > 0 && (
```

Comment:
data from url, granted verified, would have indices and not the values.

Reply:
handle, or normalize at right place with `indicesToTags`

Reply:
type is incorrect then